docs/rules/no-window-during-ssr.md

```diff
@@ -0,0 +1,34 @@
+# Disallow references to window during SSR (`lwc/no-window-during-ssr`)
```

Comment:
What is the value of the `lwc/no-window-during-ssr` rule if the `lwc/no-restricted-browser-globals-during-ssr` exists? IMO, we should consolidate all the global property access under a single rule that can be configured.

Reply:
Removed `lwc/no-window-during-ssr`.